src/source/reference/get-metrics-report.rst

```diff
@@ -49,6 +67,37 @@ data                        see below                   array       The report d
 Data contents
 -------------
 
+V2
```

Comment:
same story on the structuring of the V1 vs V2. Right now it looks intertwined and confusing as hell

Reply:
Why do you duplicate so much, why not just point out the different fields for now, so API clients can easily find what they need to change rather than re-reading all parameters and manually looking for diffs. 
At a later stage when we deprecate and drop V1 docs, we can unify.

Reply:
Personally i find it more readable when i am reading versioned docs when i dont need to do any calculations, what i mean is when i see version specific docs for the version that i am interesteed.

Reply:
Can add Version column like i did in the reporting but personally i prefer this one better